Microsoft.HealthVault.Fhir/Transformers/ImmunizationToFhir.cs

```diff
@@ -72,6 +72,17 @@ internal static Hl7.Fhir.Model.Immunization ToFhirInternal(ItemTypes.Immunizatio
             }
 
             fhirImmunization.Site = hvImmunization.AnatomicSurface.ToFhir();
+
+            fhirImmunization.Note.Add(new Hl7.Fhir.Model.Annotation {
+                Text = hvImmunization.CommonData.Note
```

Comment:
Null check before adding.

Reply:
This is not required since there is a built-in null check implemented in the library (Hl7.Fhir.Model) for most of the collections. If you look at it, the code for Note property is as the following:
 public List<Hl7.Fhir.Model.Annotation> Note
        {
            get { if(_Note==null) _Note = new List<Hl7.Fhir.Model.Annotation>(); return _Note; }
            set { _Note = value; OnPropertyChanged("Note"); }
        }